src/index.ts

```diff
@@ -1,6 +1,9 @@
 import express, {application} from 'express';
 import { connectDB } from './config/db.js';
-import {Static_Mates_Router} from "./routes/static_mates.route.js"
+import {Static_Mates_Router} from "./routes/static_mates.routes.js"
+import {Loot_Table_Router} from "./routes/loot_table.routes.js"
+import {Floor_Router} from "./routes/raid_floor.routes.js"
+//import { register } from 'node.module';
```

Comment:
Remove comment if you're not using import, it wastes memory and compilation

Reply:
Does that apply to comments in general? I like keeping things just incase we migt need them and for more note reasons but moving forward ill move them over over to notion.